company/connect-to-company.md

```diff
@@ -4,6 +4,34 @@ description: How to connect to another company
 
 # Connect to another company
 
-Explanation
+You can request connection to another company:
 
-API
+## Request
+
+{% api-method method="post" host="https://api.accp.tradecloud1.com/v2" path="/company/:companyId/connection/:otherCompanyId/_request"%} 
+{% api-method-summary %} Request company connection {% endapi-method-summary %}
+{% api-method-spec %} 
+{% api-method-request %} 
+{% api-method-headers %} 
+{% api-method-parameter name="Authorization" type="string" required=true %} Authentication token {% endapi-method-parameter %} 
```

Comment:
Value is: `Bearer <Authentication token>`
Change in all pages

Reply:
Fixed